### ISSUE TITLE
getting No module named requests error

### ISSUE THREAD

**Issue by atomicpapa** [MycroftAI/docker-mycroft#49]
When I bin/bash into the container and run `./start-mycroft.sh cli`, I am getting the following error

```
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.6/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/opt/mycroft/mycroft/__init__.py", line 17, in <module>
    from mycroft.api import Api
  File "/opt/mycroft/mycroft/api/__init__.py", line 18, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

Running latest version pulled from DockerHub tonight.

**Comment by KathyReid** [comment]
Hi there @atomicpapa, thanks for reporting this. 
I had someone on our Chat - https://chat.mycroft.ai - report something similar earlier today. 

The diagnostics we did there indicated that the Python environment they were running had version `2.21.0` of `requests`, but Mycroft requires version `2.20.0`. 

Can you do `pip3 list | grep requests` to see which version of `requests` you have installed?

**Comment by atomicpapa** [comment]
inside the container I am getting an error that pip is not found

outside the container I am running requests 2.9.1

**Comment by atomicpapa** [comment]
I upgraded requests outside the container to 2.20.0 and I am still getting the error, so it has to be inside the container and pip doesn't exist inside the container

**Comment by atomicpapa** [comment]
Managed to get reuests 2.20.0 installed but then it errored out on inflection, psutil and dateutil.  First 2 installed fine but...
```ModuleNotFoundError: No module named 'dateutil'
root@225ef0207844:/opt/mycroft# pip3 install dateutil
Collecting dateutil
  Could not find a version that satisfies the requirement dateutil (from versions: )
No matching distribution found for dateutil```

**Comment by forslund** [comment]
Hi, my guess is that either the virtualenv isn't started before running the startup-script or the requirements aren't being installed properly. I'll try an install later and see what I can do to help.

**Comment by didi-home** [comment]
Hi there,
I wanted to use the docker container for my first Mycroft-tryout :-) But I got the same error message.

However, I managed to get the CLI working (./start-mycroft.sh cli) inside the container by changing the ./start-mycroft.sh script (*)
As @forslund suggested, the venv is not started because inside the script is a check for "/.dockerenv" that prevents it.
I also noticed that the current Dockerfile has a RUN at the end that explicitly starts the venv, but the ENTRYPOINT does not. 

(*) = I created my own container with a Dockerfile
```
FROM mycroftai/docker-mycroft:latest
RUN sed -i 's/dockerenv/dockerenvnot/g' /opt/mycroft/start-mycroft.sh
```
I'm still not sure if everything works, but now it starts 4 python services at least :-)

**Comment by skliarie** [comment]
Any plans to produce fixed docker container?

**Comment by abowerman** [comment]
Not sure what the root issues are but I *was* able to get setup by running the following on the container:

`./dev_setup.sh --allow-root`
`chmod +x ./venv-activate.sh`
`source ./venv-activate.sh`
`./start-mycroft.sh all`
`./start-mycroft.sh cli`

**Comment by TheLastProject** [comment]
This should've been fixed in b18034cf85931d0d70c51e02a7b7d785b2cc5305, which is currently pending a push to Docker Hub. Please reopen if I'm wrong.
